Sources/SwiftDocC/Infrastructure/Diagnostics/DiagnosticEngine.swift

```diff
@@ -42,8 +45,9 @@ public final class DiagnosticEngine {
     }
 
     /// Creates a new diagnostic engine instance with no consumers.
-    public init(filterLevel: DiagnosticSeverity = .warning) {
+    public init(filterLevel: DiagnosticSeverity = .warning, treatWarningsAsErrors: Bool = false) {
```

Comment:
I don't remember where we landed on question of ABI stability. This would be an ABI breaking change even if the call site remains unchanged. @franklinsch is that an issue?

Reply:
If we decide not change the ABI stabilit, should it mean that we use the following code👇
```swift
public init(filterLevel: DiagnosticSeverity = .warning) {
	self.init(filterLevel: filterLevel, treatWarningsAsErrors: false)
}
public init(filterLevel: DiagnosticSeverity = .warning, treatWarningsAsErrors: Bool) {
	...
}
```

Reply:
ABI stability isn't a concern here; we don't have a library evolution workflow at the moment.

Reply:
So I guess we could leave it as it is.